packages/e2e-test-utils/src/widgets.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import { activatePlugin } from './activate-plugin';
+import { deactivatePlugin } from './deactivate-plugin';
+import { visitAdminPage } from './visit-admin-page';
+
+/**
+ * Delete all the widgets in the widgets screen.
+ */
+export async function cleanupWidgets() {
```

Comment:
I'm not really sure this should be exposed in this package 🤔

Reply:
TBH, I'm not sure either. It's only relevant to the widgets editor, but on the other hand, most of the utils in this package are only relevant to the post editor too.

I'm open to any other suggestions though.

Reply:
@ntsekouras Out of interest, what are the reasons?

The only comments I'd have is that the name could maybe be something like `deleteAllWidgets`, 'cleanup' is a bit ambiguous.

The implementation could probably change to use the block-based widget editor to delete all widgets at some point.

Other than that, if we're unsure about it in the test utils package, it could be made unstable.

Reply:
IMO it feels like it could live inside the `widgets` tests a bit more - on the other hand widgets are getting into core now... @youknowriad any thoughts?

Reply:
> IMO it feels like it could live inside the widgets tests a bit more 

I actually tried that. But the jest config will tries to run any JS files inside the `specs` folder. As I can see, we have 4 options:

1. Move it to `e2e-test-utils` (this PR)
2. Inline them
3. Update the jest config to only look for files end with `.test.[tj]s`
4. Add `__helpers__` (or any other names) to `testPathIgnorePatterns` in the jest config, and place it into that folder.

> The only comments I'd have is that the name could maybe be something like deleteAllWidgets, 'cleanup' is a bit ambiguous.

I can change that!

Reply:
GH didn't refreshed and didn't see your response @talldan :) 

> Out of interest, what are the reasons?

My initial thought was about the widgets not being in core, but now that they are getting in, I guess it's okay, yes.

Reply:
I don't think it's outside the remit of the package, which just considers this WordPress utils:
https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-test-utils/README.md#e2e-test-utils

But open to different opinions from other maintainers.

There are some similar general WordPress utils, I noticed a `deleteTheme` function. And there's `trashAllPosts`:
https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-test-utils/src/posts.js#L21

Thanks for renaming it @kevin940726, I think that brings more consistency. 👍

Reply:
> GH didn't refreshed and didn't see your response @talldan :)

I didn't see the other comments either till much later 😄